package.json

```
@@ -9,8 +9,8 @@
     "prepare": "npm run lint && npm run build",
     "clean": "rm -rf dist",
     "build": "npm run clean && tsc --declaration",
-    "test": "jest",
-    "test:watch": "jest --watch",
+    "test": "vitest run",
+    "test:watch": "vitest",
```

If we change this to:

```suggestion
    "test": "npx vitest run",
    "test:watch": "npx vitest",
```

It will ensure that the version of vitest from this repo is used, instead of the globally installed version. (This is also nice for people who don't have `vitest` installed globally.)

---

This isn't needed. You don't need to have `vitest` installed globally for a call to `vitest` in `scripts` to work. https://vitest.dev/guide/#adding-vitest-to-your-project

---

I checked out this branch and ran `npm run test`, and it complained about `vitest` not being globally installed:

```
> simple-machines@0.5.0 test
> vitest run

sh: vitest: command not found
Exit code: 127
```

---

Oh! Never mind. I needed to run `npm install` first. I think we're good here.